recipes/urbansim_templates/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "urbansim_templates" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "a2df420eb85fcfc8d37a4a027217c4254e2d946626ca6c84baf006f2b045acda" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - choicemodels >= 0.2
+    - numpy >= 1.14
+    - orca >= 1.4
+    - pandas >= 0.22
+    - patsy >= 0.4
+    - python
+    - statsmodels >= 0.8
+    - urbansim >= 3.1
```

Comment:
The syntax for these is unfortunately incorrect. There should be no space after the `>=` (only before). So it should be `urbansim >=3.1`. This applies to the rest as well. Mentioning this as the recipe is [failing to convert]( https://travis-ci.org/conda-forge/staged-recipes/builds/489687358 ) due to this syntax error. Could you please send a PR to fix the recipe, @smmaurer?

Reply:
@jakirkham Sorry, thanks so much for pointing this out! I just opened a new PR (#7734) that fixes the syntax.